pkg/chains/formats/slsa/attest/attest.go

```diff
@@ -58,10 +59,11 @@ func Step(step *v1beta1.Step, stepState *v1beta1.StepState) StepAttestation {
 	return attestation
 }
 
-func Invocation(source *v1beta1.RefSource, params []v1beta1.Param, paramSpecs []v1beta1.ParamSpec, meta metav1.Object) slsa.ProvenanceInvocation {
+func Invocation(obj objects.TektonObject, params []v1beta1.Param, paramSpecs []v1beta1.ParamSpec, meta metav1.Object) slsa.ProvenanceInvocation {
```

Comment:
Seems like there are some improvement area for this function interface though I understand that we do need to separate out params and paramSpec to accept both task level and pipeline level. 

Maybe remove `meta metav1.Object` b/c `obj` already contains meta data right?

Reply:
Yes. Fixed. Thanks!